install/kubernetes/cilium/charts/operator/templates/clusterrole.yaml

```diff
@@ -65,3 +66,17 @@ rules:
   - leases
   verbs:
   - '*'
+{{- end }}
+{{- if and (lt .Capabilities.KubeVersion.Minor "14") (eq .Capabilities.KubeVersion.Major "1") }}
+- apiGroups:
+  - ""
+  resources:
+  # To be used when operator HA mode fallbacks to the use of ConfigMapLock
+  # instead of LeasesLock due to lack of support for coordination.k8s.io
+  # API Group.
+  - configmaps
+  verbs:
+  - create
+  - get
+  - update
+{{- end }}
```

Comment:
needs a new line

Reply:
you can put an `{{ else }}` here and use everything from line 60 to 68 in this else statement